src/dmd/func.d

```diff
@@ -198,6 +198,22 @@ extern (C++) struct Ensure
  */
 extern (C++) class FuncDeclaration : Declaration
 {
+    /// All hidden parameters bundled.
+    static struct HiddenParameters
+    {
+        /**
+         * The `this` parameter for methods or nested functions.
+         *
+         * For methods, it would be the class object or struct value the
+         * method is called on. For nested functions it would be the enclosing
+         * function's stack frame.
+         */
+        VarDeclaration this_;
```

Comment:
Why this_ instead of vthis (as used elsewhere)?

Reply:
And how is it different from .vthis?

Reply:
It is the same as `vthis`, I can renamed it but I think `this_` is a better name and follows D naming conventions.

Reply:
The convention for a variable representing the `this` value is `vthis`. This appears in the front end 146 times.

Reply:
Doesn't mean it's a good name, but I'll rename it.

Reply:
Renamed.